README.md

```diff
@@ -14,8 +14,6 @@ The Charmed Operator can be found on [Charmhub](https://charmhub.io/kafka-k8s) a
 - SASL/SCRAM auth for Broker-Broker and Client-Broker authentication enabled by default.
 - Access control management supported with user-provided ACL lists.
 
-The Charmed Kafka K8s Operator uses Kafka binaries released by the The Apache Software Foundation, made available using the [`ubuntu/kafka` OCI image](https://registry.hub.docker.com/r/ubuntu/kafka) distributed by Canonical.
```

Comment:
Why are we deleting that from the readme file? In a PR called "Adding references...". I feel like this information is more visible here, rather than in metadata.yaml.

Reply:
It is a typo, and it is not really correct since: 
1. We are packing our own binaries
2. The images are not hosted to docker hub anymore